test/run_envoy_bazel_coverage.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e -o pipefail
 
-LLVM_VERSION=${LLVM_VERSION:-"14.0.0"}
+LLVM_VERSION=${LLVM_VERSION:-"18.1.8"}
```

Comment:
It's just a nit, but maybe we can make the Envoy CI docker image expose LLVM_VERSION that it comes with. And when it's not provided rely on llvm-config to get the default value (i.e., use `llvm-config --version` here instead of hardcoding a particular version, similarly to what is done below for clang, llvm-cov, etc)?

Reply:
yeah - ive never been comfortable with this check - not sure why it was added but my guess is that a realization that coverage can be different with different llvm versions - lets deal with this one separately